docs/src/pages/index.soy

```diff
@@ -108,7 +108,7 @@ description: "A collection of well known Algebraic Data Types for your utter enj
       <section class="highlight row">
         <div class="col-md-6 col-md-push-9">
           <h4 class="highlight-title">Use a collection of popular Algebraic Data Types (ADTs)</h4>
-          <p class="highlight-description">You have heard of things like Maybe and Either and heck maybe even IO, that is what Crocks provide.</p>
+          <p class="highlight-description">Have you ever heard of Maybe, Either, or, heck, even IO? These are a few ADTs which Crocks brings to the table.</p>
```

Comment:
I like this direction and qualms about it reading like:

"These are just a few of the ADTs which Crocks brings to the table."

Reply:
Sorry I didn't get notified 8 hours ago!  I wish there was a in-browser editor for times like these.